src/ex4/server/DB/pokemonsCache.json

```diff
@@ -1 +1 @@
-[{"time":1655387608179,"input":"2","response":["Catch ivysaur the grass/poison pokemon"]}]
\ No newline at end of file
+{"12":["Catch butterfree the bug/flying pokemon"],"15":["Catch beedrill the bug/poison pokemon"],"27":["Catch sandshrew the ground pokemon"],"time":1656002659494,"15,86":["Catch beedrill the bug/poison pokemon","Catch seel the water pokemon"]}
```

Comment:
better not to upload files with data to github, especially if they are specific user data